composer.json

```diff
@@ -126,6 +126,7 @@
         "drupal/whatlinkshere": "^1.0",
         "drush/drush": "^9.0.0",
         "harvesthq/chosen": "^1.8",
+        "symfony/event-dispatcher": "4.3.4 as 3.4.35",
```

Comment:
NB: this will need to come out once Core + search API modules can reconcile their differences (patch is RTBC for Core to permit search API to cut a new release).